server/src/routes/api/v1/ingredientsRouter.js

```diff
@@ -53,4 +71,16 @@ ingredientsRouter.post("/", async (req, res) => {
   }
 })
 
+ingredientsRouter.delete("/", async (req, res) => {
+  const { ingredientId } = req.body
+  const user = req.user
+  try {
+    const existingMeasurement = await user.$relatedQuery("pantryMeasurements").where("ingredientId", ingredientId).first()
+    const deleted = await PantryMeasurement.query().deleteById(existingMeasurement.id)
```

Comment:
Im curious if there is any chance that a user has multiple PantryMeasurements? like add butter to the pantry twice? 
Even if that isnt the case, you should be able to avoid having to call `first()` on line 78 and just delete ALL PantryMeasurements that would be returned from that relatedQuery